### DISCUSSION TITLE
Is it possible to install with disk encryption?

### DISCUSSION THREAD

**Discussion by me-self** [itsManjeet/avyos#216] [Q&A] [answered]
Hello! :wave: 
I was just wondering whether or not it is possible to install with disk encryption.
Although I didn't notice such an option with the full disk install option, I'm wondering if it is supported through manual partitioning?

**Comment by itsManjeet** [comment, marked as answer]
Hey,
disk encryption is currently not possible with the default rlxos installation, and not planned yet.

**Reply by Darkle** [reply]
Is home folder encryption planned?

**Reply by itsManjeet** [reply]
Not In the Next release, which is primarily focused on better Application (or Package Management) Support.
Sorry for Inconvenience.

**Comment by DEVHELP24** [comment]
Please dont forget some distros cant use full disk encryptions!!!!!!!!!!!!!!!!!!!!!!!!!!
